GDJS/Runtime/types/project-data.d.ts

```diff
@@ -317,6 +317,9 @@ declare interface ProjectPropertiesData {
   author: string;
   authorIds: string[];
   authorUsernames: string[];
+  playerId: string;
```

Comment:
No need to store this in the project properties data? This is not in the JSON right?

Reply:
Indeed, this can be removed